src/containers/bitset_containers.c

```diff
@@ -0,0 +1 @@
+#include <../src/bitset.c>
```

Comment:
Should this actually be the local bitset.c file in the containers folder? `#include "bitset.c"`